src/docs/product/performance/transaction-summary.mdx

```diff
@@ -80,6 +91,29 @@ The **Tags** tab displays a list of suspect tag keys that often correspond to sl
 
 The table in **All Events** tab offers a full list of transactions broken down similar to the abbreviated table in the overview. Filters are carried across from other views. You can also narrow the events down by percentile.
 
+## Spans
+
+<Note>
+
+This feature is available only if you're in the Early Adopter program. Features available to Early Adopters are still in-progress and may have bugs. We recognize the irony.
+If you’re interested in being an Early Adopter, you can turn your organization’s Early Adopter status on/off in General Settings. This will affect all users in your organization and can be turned back off just as easily.
+
+</Note>
+
+The **Spans** tab displays a list of suspect spans that corresponds to where most of the time is being spent. Choose from one of the metrics to identify different types of problems and filter down to the span operation you are interested in to find spans where performance problems lie. Example events are available and will take you directly to the span in question.
+
+Suspect spans are determined using a span's exclusive time rather than its duration. The exclusive time of a span is the time spent in a span excluding the time spend in any child spans.
```

Comment:
```suggestion
Suspect spans are determined using a span's exclusive time rather than its duration. The exclusive time of a span is the time spent in a span excluding the time spent in any child spans.
```